packages/types/src/extrinsic/types.ts

```diff
@@ -90,6 +91,129 @@ export interface SignatureOptions extends SignatureOptionsBase {
   doughnut?: AnyU8a | Doughnut;
   transactionPayment?: AnyU8a | ChargeTransactionPayment;
 }
+const AllHashers = {
```

Comment:
How about relocate these changes into separated files, so that we know they are types for metadata rather for extrinsic, easier for further refactoring and maintaining?

Reply:
sure.. will do that..